examples/c/uprobe.c

```diff
@@ -80,11 +80,14 @@ int main(int argc, char **argv)
 	uprobe_offset = get_uprobe_offset(&uprobed_add);
 
 	/* Attach tracepoint handler */
+	// clang-format off
```

Comment:
well, this sucks, let's not add clang-format off in the middle of the code. What was the suggested formatting?